libnmstate/nm/connection.py

```diff
@@ -168,8 +168,9 @@ def safe_activate_async(self):
             user_data,
         )
 
-    def _active_connection_callback(self, src_object, result, user_data):
-        cancellable = user_data
+    @staticmethod
+    def _active_connection_callback(src_object, result, user_data):
```

Comment:
This does not make sense. The callback should call correctly.
See example:
```
>>> class A(object):                      
...     def foo(self, arg):               
...         print(self, arg)              
>>> def call_func(f):                     
...     f(1)                              
>>> call_func(A().foo)                    
(<__main__.A object at 0x7fc0354c6a50>, 1)
```

Reply:
Your example does not show the object is newly created or old one.

Check this:

```
In [1]: class A(object):
   ...:     def foo(self, arg):
   ...:         print(self, arg)
   ...:

In [2]: a = A()

In [3]: a.abc =1

In [4]: def call_func(f):
   ...:     f(1)
   ...:

In [5]: a
Out[5]: <__main__.A at 0x7f620a045f10>

In [6]: call_func(A().foo)
<__main__.A object at 0x7f62085df3d0> 1
```